blocks/editable/format-toolbar.js

```diff
@@ -151,17 +152,24 @@ class FormatToolbar extends Component {
 						className="editable-format-toolbar__link-modal"
 						style={ linkStyle }
 						onSubmit={ this.submitLink }>
+						<label
```

Comment:
I have't found a resource to list what the current assistive technology software is supporting, and whether aria-label functions in a similar way to the visually hidden label.

The aria is the "W3C" way to handle such a scenario.

I've just tested VoiceOver on the mac and it reads out loud the aria-label, just like it does with the hidden label. 

I've looked at a survey from 2 years ago:
http://webaim.org/projects/screenreadersurvey6/
and the charts were lead by commercial screen reader software applications - Jaws / ZoomText and Window Eyes.
Free solutions like NVDA and VoiceOver had a lower usage percentage. 

And if this worked in VoiceOver, I would say that the commercial ones should support this just as well.

Really good input on this, @aduth !